src/engine/threaded_engine_perdevice.cc

```diff
@@ -114,7 +114,7 @@ class ThreadedEnginePerDevice : public ThreadedEngine {
           auto ptr =
           gpu_copy_workers_.Get(ctx.dev_id, [this, ctx, is_copy, nthread]() {
             // Signify to kernel that GPU is being used,  no Kernel Launch OMP (temporary behavior)
-            Engine::Get()->set_num_omp_threads_per_worker(1);
+            OpenMP::Get()->set_enabled(false);
```

Comment:
For now, just "switch it off", which will just return '1' for number of OMP threads whenever OpenMP class is asked.